pkg/gds/admin.go

```diff
@@ -561,7 +561,10 @@ func (s *Admin) acceptRegistration(vasp *pb.VASP) (msg string, err error) {
 		return "", err
 	}
 	vasp.VerifiedOn = time.Now().Format(time.RFC3339)
-	vasp.VerificationStatus = pb.VerificationState_REVIEWED
+	// TODO: Replace "email" in source parameter with user email address.
```

Comment:
I'm working on the authentication middleware right now that will add the user email to the request context. Would you prefer to hold off on finishing this PR until that's merged and then rebase with it to implement it, or create a story to do this later and merge this PR without waiting for the middleware?

Reply:
Seems like a simple enough change so I'll hold off on merging this PR until the middleware one is done.